src/ui/FieldValue/FieldValue.ts

```diff
@@ -362,6 +362,7 @@ export class FieldValue extends Component {
   private getHelperOptions() {
     const inlineOptions = ComponentOptions.loadStringOption(this.element, 'helperOptions', {});
     if (Utils.isNonEmptyString(inlineOptions)) {
+      // Eval
       return extend({}, this.options.helperOptions, eval('(' + inlineOptions + ')'));
```

Comment:
An `eval` is used here to convert a string object specified in html to a JS object. To remove this `eval` , we could require the html config to be valid JSON, and then use `JSON.parse`.

The supported options are strings, booleans, and numbers (i.e. values that can be serialized as JSON). I think this is a reasonable breaking change to make, and can log a message to the console when we detect invalid JSON.